### PR TITLE
win: Fix the inconsistent usage of dllexport/dllimport

### PR DESCRIPTION
**Pull request by kaadam**
Set the proper prefix for the delcarations.

### CODE REVIEW

**Review by bryanpkc — APPROVED** [review]

File: runtime/include/FuncArgMacros.h [hunk 1]
```diff
@@ -127,13 +126,14 @@
...
 
-/* #if defined(WIN64) || defined(WIN32) */
-#if defined(PGDLL) && defined(_DLL) &&                                         (defined(TARGET_WIN) || defined(WIN64) || defined(WIN32))
-#define WIN_EXP __declspec(dllexport)
-#define WIN_IMP extern __declspec(dllimport)
+#if defined(_WIN64)
+  #if defined(_DLL)
+   #define WIN_API extern __declspec(dllexport)
+ #else
+   #define WIN_API extern __declspec(dllimport)
```
[the remark this answers is not in the document]
Comment (isuruf) [reply runtime/include/FuncArgMacros.h:133]: I think this should be removed avoid messing up the static runtime library.

[the remark this answers is not in the document]
Comment (kiranchandramohan) [reply runtime/include/FuncArgMacros.h:133]: ping @kaadam. Is the issue that @isuruf pointing out addressed?

[the remark this answers is not in the document]
Comment (kaadam) [reply runtime/include/FuncArgMacros.h]: @isuruf updated the change, I removed dllimport.

**Review by isuruf — APPROVED** [review]

**Comment by kaadam** [comment]
@bryanpkc @pawosm-arm @kiranchandramohan  Could you take a look at it please?

**Pull request merged**